pyproject.toml

```diff
@@ -8,7 +8,7 @@ version = "0.1.0"
 authors = [
   {name = "DrivenData", email = "info@drivendata.org"}
 ]
-description = "CyFi is a Python package to estimate cyanobacteria density in small, inland water bodies using Sentinel-2 satellite imagery."
+description = "Estimate cyanobacteria density in small, inland water bodies using Sentinel-2 satellite imagery."
 readme = "README.md"
 license = {text = "MIT License"}
```

Comment:
I think we want to point at the license file here with `license = { file = "LICENSE" }`